data/project-templates/js-node/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "nodejs-golem-app",
+  "version": "1.0.0",
+  "description": "NodeJS script using Go",
```

Comment:
this is probably a wrong description...?

Reply:
Uncompleted :)
`Should be NodeJS script using Golem Network`